libpebble2/util/bundle.py

```diff
@@ -40,6 +40,7 @@ class PebbleBundle(object):
         'basalt': ('basalt/', ''),
         'chalk': ('chalk/',),
         'diorite': ('diorite/', 'aplite/', ''),
+        'emery': ('emery/', 'basalt/', 'aplite/', ''),
```

Comment:
`aplite/` isn't valid, since we don't support backwards compatibility with SDK 3 aplite apps.